### DISCUSSION TITLE
GIthub Markdown has Footlinks

### DISCUSSION THREAD

**Discussion by thejeshgn** [zim-desktop-wiki/zim-desktop-wiki#1647] [Ideas]
[Footnotes now supported in Markdown fields](https://github.blog/changelog/2021-09-30-footnotes-now-supported-in-markdown-fields/)

We already have inside page links in ZimWki. A modified version of it which works as footlinks will be great. I currently use special anchor `"##name"` for doing footlinks.

```
==== Projects ====
 {{id: ref-1}} go to  footnote [[[#footnote-1|1]]] 

==== Footnotes ====
1. {{id: footnote-1}} This is footnote[[[#ref-1|1]]]  

```

It works well. But its a little cumbersome.  I was wondering If I can write a plugin that will do this automagically. 

--

This is the github way of doing a simple footnote[^1]. With some additional text after it.

[^1]: My reference.

**Comment by Nick222** [comment]
Let it be special small window for read footnotes...
